metabase/metabase
# feat(sdk): make editable dashboard grid border color themeable

Closes https://github.com/metabase/metabase/issues/47567

### Description

Updates the editable dashboard's dashboard grid border color to be themeable.

Dashboard grid colors will use the border color in `colors.border` by default, and allows overriding with `components.dashboard.gridBorderColor` if you want the grid color to differ than other border colors (useful because in some themes you want the grid border to be less noticeable)

### How to verify

- Go to the `EditableDashboard` folder in Storybook.
- Check out the "Dark Theme" story and click on edit. The border color should now come from `color.borders` from the story.
- Check out the "Custom Grid Color" story and click on edit. The border color should now come from `components.dashboard.gridBorderColor` from the story.
